.travis.yml

```diff
@@ -9,13 +9,20 @@ jdk:
 stages:
   - test
   - build
+  - name: docker
+    if: branch = master && type != pull_request
```

Comment:
I don't think this runs for releases, which are tag builds and do not have a branch